pubsublite/consumer.go

```diff
@@ -163,8 +168,13 @@ func NewConsumer(ctx context.Context, cfg ConsumerConfig) (*Consumer, error) {
 			return nil // nil is returned to avoid terminating the subscriber.
 		},
 	}
-	consumers := make([]consumer, 0, len(cfg.Subscriptions))
-	for _, subscription := range cfg.Subscriptions {
+	consumers := make([]consumer, 0, len(cfg.Topics))
+	for _, topic := range cfg.Topics {
+		subscription := Subscription{
+			Name:    string(topic),
+			Project: cfg.Project,
+			Region:  cfg.Region,
+		}
 		client, err := pscompat.NewSubscriberClientWithSettings(
 			ctx, subscription.String(), settings, cfg.ClientOpts...,
 		)
```

Comment:
Should we change the `logger` and `telemetryAttributes` down below to use the data from cfg rather than the subscription?

Reply:
Seems equivalent to me. Do you see a benefit in doing that?

Reply:
It is indeed equivalent. The only benefit I would see is in the ease of reading/understanding. But that's very subjective.

Reply:
Pushed https://github.com/elastic/apm-queue/pull/119/commits/4fb881b6af890a664aeb25d548cf765762589c1d, is that what you had in mind? I don't personally see an improvement, but if you find this easier to read and understand then that works for me :)